decidim-proposals/app/controllers/decidim/proposals/proposals_controller.rb

```diff
@@ -232,7 +231,7 @@ def default_filter_origin_params
       def default_filter_category_params
         return "all" unless current_component.participatory_space.categories.any?
 
-        ["all"] + current_component.participatory_space.categories.map { |category| category.id.to_s }
+        ["all"] + current_component.participatory_space.categories.pluck(:id)
```

Comment:
To keep the original behavior (just in case):

```suggestion
        ["all"] + current_component.participatory_space.categories.pluck(:id).map(&:to_s)
```

Reply:
I did test the change locally to check it worked, but anyway, you may be right. Let's may be sure we don't have unexpected crashes.

Reply:
Yeah, I supposed it wouldn't have crashes but I think it's best if we're safe and keep the behavior 😄